<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncomeExpense</class>
 <widget class="QDialog" name="IncomeExpense">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IncomeExpense</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,10,10,1">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>İşlemler</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1,1,2,1,1,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Başlangıç Tarihi </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEditDateStart">
            <property name="date">
             <date>
              <year>2015</year>
              <month>3</month>
              <day>19</day>
             </date>
            </property>
            <property name="displayFormat">
             <string>yyyy.MM.dd</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_23">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Bitiş Tarihi </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEditDateEnd">
            <property name="date">
             <date>
              <year>2015</year>
              <month>3</month>
              <day>19</day>
             </date>
            </property>
            <property name="displayFormat">
             <string>yyyy.MM.dd</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Türü</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxPayType">
            <item>
             <property name="text">
              <string>Hepsi</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Aidat</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Şartlı Bağış</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonQueryWithDate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string> Sorgula</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources/bys.qrc">
            <normaloff>:/Fotos/play.png</normaloff>:/Fotos/play.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonShowAll">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Tümü</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources/bys.qrc">
            <normaloff>:/Fotos/play.png</normaloff>:/Fotos/play.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonEnterExpense">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>    Gider Ekle</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources/bys.qrc">
            <normaloff>:/Fotos/play.png</normaloff>:/Fotos/play.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonExportPDF">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string> PDF Aktar</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources/bys.qrc">
            <normaloff>:/Fotos/PDF.ico</normaloff>:/Fotos/PDF.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Gelirler</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableViewIncome">
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelSumIncome">
          <property name="text">
           <string>Gelirler Toplamı : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTotalIncomes">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Giderler</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="10,0">
      <item>
       <widget class="QTableView" name="tableViewExpenses">
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelSumOut">
          <property name="text">
           <string>Giderler Toplamı : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTotalExpenses">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelLast">
       <property name="text">
        <string>Kasa Net  : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTotalKasa">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/bys.qrc"/>
 </resources>
 <connections/>
</ui>
